eox_nelp/migrations/0007_auto_20240522_2251.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.21 on 2024-05-22 22:51
```

Comment:
Could you change the name of the migration to something more telling(verbose)

Reply:
Done, just keep in mind that those files are autogenerated, the name is the django standard, in this case  this depends on [this](https://github.com/eduNEXT/eox-nelp/pull/166)  so the new migration has a better name